src/Verify/Serialization/Scrubbers/ApplyScrubbers.cs

```diff
@@ -65,9 +67,12 @@ static void AddProjectAndSolutionReplacements(string? solutionDir, string projec
         {
             return;
         }
+
+        var regex = new Regex(@"([^/\\])[/\\]+$");
```

Comment:
can u do this without a regex?

Reply:
I think so, what are the concerns related to regex?

Reply:
regex is slower, uses more memory, and more difficult to read than standard string manipulation

Reply:
Okay, I will rewrite the logic to use common string manipulations.